requirements.txt

```diff
@@ -1,3 +1,3 @@
-Django==3.2.5
 beautifulsoup4==4.9.1
+Django==4.0.1
```

Comment:
`4.0.1` can only be used for py38 and above. As a result, we can still see `3.2.5` being used in py36 below (see `pip_dependencies` field below.

Suggestion: I find `3.2.12` to pass the ProdScan and work on py36+

Otherwise, we can use `4.0.1` on py39 and `3.2.12` on py36. I just though `3.2.12` looks simpler.

Reply:
Make sense. Changed the Django version